lib/v2/dut/shortcuts.js

```diff
@@ -35,13 +35,34 @@ module.exports = {
         tzgg: 'tzgg/tzgg',
         xqxw: 'xqxw/xqxw',
     },
+    panjin: {
+        xqxw: 'index/xqxw',
+        xywh: 'index/xywh',
+        xyfc: 'index/xyfc',
+        xsdt: 'index/xsdt',
+        mtbd: 'index/mtbd',
+        gg: 'index/gg',
```

Comment:
https://panjin.dlut.edu.cn/index/tzgg.htm
```suggestion
        tzgg: 'index/tzgg',
        gg: 'index/gg',
```